timApp/static/scripts/tim/gamification/badge/badge.component.ts

```diff
@@ -0,0 +1,75 @@
+import type {OnInit, OnChanges} from "@angular/core";
+import {Component, Input, NgModule, SimpleChanges} from "@angular/core";
+import {CommonModule} from "@angular/common";
+import {FormsModule} from "@angular/forms";
+
+@Component({
+    selector: "tim-badge",
+    template: `
+    <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" rel="stylesheet" />
```

Comment:
Vois katsoa saako näitä fonttitiedostoja kopioitua suoraan repoon niin että kaikki ikonit olis hostattuna itse timissä